script/sync-crls

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# script/sync-crls: update the DOD CRL bundle and place it where authnid expects them
+
+rm -rf crl ALLCRLZIP.zip || true
```

Comment:
Should the `rm ALLCRLZIP.zip` part be done at the end of this script? Is it necessary for that file to hang around after unzipping and copying the CRLs?

Reply:
Nope, you're right. It is in the `gitignore` but there's no reason to keep it around.